LibCore/CProcessHelper.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <AK/AKString.h>
+#include <AK/HashMap.h>
+#include <AK/RetainPtr.h>
+
+class CProcessInfo : public Retainable<CProcessInfo> {
+public:
+    pid_t pid;
+    unsigned nsched;
+    String name;
+    String state;
+    String username;
+    uid_t uid;
+    String priority;
+    size_t linear;
+    size_t physical;
+    unsigned syscalls;
+};
+
+class CProcessHelper {
+public:
+    CProcessHelper();
+    HashMap<pid_t, RetainPtr<CProcessInfo>> get_map();
```

Comment:
If you make the above change, this would simply be:
HashMap<pid_t, CProcessStatistics> get_map();